contrib/devtools/lint-python-shebang.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+EXIT_CODE=0
+for PYTHON_FILE in $(git ls-files -- "*.py"); do
+    if [[ $(head -c 2 "${PYTHON_FILE}") == "#!" &&
+          $(head -n 1 "${PYTHON_FILE}") != "#!/usr/bin/env python3" ]]; then
+        echo "Missing shebang \"#!/usr/bin/env python3\" in ${PYTHON_FILE}"
```

Comment:
Comment could be clearer. Something like "Shebang must use python3 (not python or python2)"